### PR TITLE
[Scheduler] Fix scheduler examples

### PR DESCRIPTION
**Pull request by alamirault**
I tried to follow documentation and found some issues

### CODE REVIEW

File: scheduler.rst [hunk 1]
```diff
@@ -105,9 +108,13 @@ The :class:`Symfony\\Component\\Scheduler\\Attribute\\AsSchedule` attribute,
 which by default references the schedule named ``default``, allows you to register
 on a particular schedule::
 
-    // src/Scheduler/MyScheduleProvider.php
+    // src/Scheduler/SaleTaskProvider.php
     namespace App\Scheduler;
 
+    use Symfony\Component\Scheduler\Attribute\AsSchedule;
```
Comment (alamirault) [inline scheduler.rst:114]: How `code-block-checker` can know theses classes ?

Is adding `symfony/scheduler` in its compoer.json will be enought ?

Comment (OskarStark) [reply scheduler.rst:114]: Yes this should be enough

**Review by OskarStark — APPROVED** [review]

File: scheduler.rst (hunk 1 above)
Comment (alamirault) [reply scheduler.rst:114]: It's in https://github.com/symfony-tools/symfony-application/pull/26 :see_no_evil:

Comment (OskarStark) [reply scheduler.rst:114]: Merged

**Comment by javiereguiluz** [comment]
Thank you Antoine.

**Pull request merged**